src/ui/src/mainwindow.cpp

```diff
@@ -1947,6 +1947,7 @@ void MainWindow::writeSettings()
     std::vector<
         std::tuple<const ViewInterface*, uint64_t, std::shared_ptr<const ViewContextInterface>>>
         widget_list;
+    widget_list.reserve( static_cast<size_t>( mainTabWidget_.count() ) );
```

Comment:
Another settings place. Let's remove reserve

Reply:
fix it https://github.com/variar/klogg/commit/a4f9cad9eceff87a667f1c6cec939f383153a6e3